gigareview/console/migrations/m220629_145450_create_manuscript_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles the creation of table `{{%manuscript}}`.
+ */
+class m220629_145450_create_manuscript_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->createTable('{{%manuscript}}', [
+            'id' => $this->primaryKey(),
+            'manuscript_number' => $this->string(),
+            'article_title' => $this->string(),
+            'revision_number' => $this->smallinteger(),
+            'created_at' => $this->bigInteger(),
+            'updated_at' => $this->bigInteger(),
```

Comment:
* `revision_number` is not required, and neither is `revison_id`
* Keep `created_at` and `updated_at` for table timestamps
* Need a `publication_date` field
* Need a `doi` field
* Need an `editorial_status_date` field
* Need an `editorial_status` field
* Update ER diagram using omnigraffle